docs/docs/conceptual/gatsby-for-ecommerce.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Using Gatsby For E-commerce
+---
+
+Businesses selling products online typically need a variety of software to support their experience. At a minimum, their website needs product pages, product catalog navigation, a shopping cart, and checkout. 
+
+Most have additional functionality like customer account creation, discount codes, customer reviews, tax calculation, user tracking via analytics, and content personalization.
+
+The website functions because these front-end capabilities integrate with a wide swath of software on the backend, such as inventory management, order fulfillment, business analytics, and customer engagement via email. 
+
+### Choosing a main e-commerce platform
+
+Most businesses choose a central e-commerce platform as their source of truth for these functionality. Some businesses choose to run entirely on these platforms, which can be quicker to get started with but lock you in to their choices for website creation (eg Liquid Templates for Shopify). 
+
+If you're using or considering Gatsby, your organization is likely optimizing for specific properties of your website (such as performance, development environment, design, user experience) and are therefore considering a "JAMstack", "decoupled" or "content mesh" architecture. 
+
+In this case, your options are likely between a technologically forward e-commerce vendor with headless capabilities, such as Shopify or BigCommerce, or a new, specialized JAMStack e-commerce vendor, like Snipcart, Nacelle, or CommerceLayer. All of these have out-of-the box Gatsby integrations.
+
+### Choosing additional content systems
+
+Once an organization selects a main e-commerce platform, it may also want or need other content stores that will get pulled into the website. They might choose a CMS like Contentful for complex content modelling, or Wordpress for blog content authoring. They might choose Yotpo for customer reviews or Salsify to store product SKUs. 
```

Comment:
I think Salsify is better described as a product information management tool, not just SKUs. It's tracks other important information.

Reply:
Good point.